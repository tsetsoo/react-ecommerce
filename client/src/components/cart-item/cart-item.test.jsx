import { shallow } from 'enzyme'
import React from 'react'
import { CartItem } from './cart-item.component'

it('expect to render CartItem component', () => {
    const fakeItem = { imageUrl: 'fake-url', price: 10, name: 'fake-item', quantity: 1 }
    expect(shallow(<CartItem item={fakeItem}/>)).toMatchSnapshot();
})