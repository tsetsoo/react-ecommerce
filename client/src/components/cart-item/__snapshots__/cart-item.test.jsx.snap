// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render CartItem component 1`] = `ShallowWrapper {}`;
